metering/metering-upgrading-metering.adoc

```diff
@@ -65,12 +65,13 @@ You can verify the metering upgrade by performing any of the following checks:
 .Procedure (CLI)
 *  Check the Metering Operator CSV:
 +
+[source,terminal]
```

Comment:
Does the code block at line 49 need markup?
https://github.com/openshift/openshift-docs/pull/24941/files#diff-a326385a73155eb101aed6cec1127881R49

Reply:
@lbarbeevargas There us a code block in step 6 that doesn't have a [source]:

```
----
Metering
4.5.0-202007012112.p0 provided by Red Hat, Inc
----
```

Reply:
Ah gotcha, thank you @mburke5678. That is an example of what you would see in the web console and not the CLI, so I left it without  a [source]. Is that the best course of action? Thank you.